.circleci/config.yml

```diff
@@ -153,16 +150,13 @@ commands:
     parameters:
       k8s_svc_name:
         type: string
-        default: "staging-deriv-com"
+        default: "deriv-com-staging"
```

Comment:
Both `k8s_deploy` and `k8s_deploy_staging` have the same set of commands we can remove the `k8s_deploy_staging` config block and re-use `k8s_deploy` with `deriv-com-staging` as parameter